libsm.yaml

```diff
@@ -0,0 +1,63 @@
+package:
+  name: libsm
+  version: 1.2.3
+  epoch: 0
+  description: X11 Session Management library
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: MIT
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - libice-dev
+      - xorgproto
+      - xtrans
+      # - util-linux-dev TODO let's do util-linux-dev later and enable the configure flag --with-libuuid below
+      - util-macros
+      - xmlto
+      - bash
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 2d264499dcb05f56438dee12a1b4b71d76736ce7ba7aa6efbf15ebb113769cbb
+      uri: https://www.x.org/releases/individual/lib/libSM-${{package.version}}.tar.bz2
+  - uses: autoconf/configure
+    with:
+      opts: |
+        --enable-docs \
+        --with-xmlto \
+        --without-fop
+
+  # --with-libuuid TODO as above, would rather not build util-linux-dev yet
```

Comment:
is this ok?  avoiding configuring `--with-libuuid` so we don't need to package  `util-linux` yet.

Reply:
Yeah it’s fine for now!